demisto_sdk/commands/create_artifacts/tests/content_creator_test.py

```diff
@@ -0,0 +1,46 @@
+import filecmp
+from tempfile import mkdtemp
+
+from demisto_sdk.commands.create_artifacts.content_creator import *
+from demisto_sdk.commands.common.git_tools import git_path
+
+
+class TestContentCreator:
+    def setup(self):
+        current_dir = os.path.normpath(os.path.join(__file__,
+                                                    f'{git_path()}/demisto_sdk/commands/create_artifacts', 'tests'))
```

Comment:
`git_path` returns absolute path to the root of the repo, no need to use the current file location.
```
current_dir = f'{git_path()}/demisto_sdk/commands/create_artifacts/tests'
```